### PR TITLE
Improve VTEX App Store Product page

### PR DESCRIPTION
**Pull request by ntmedina**
## Markdown (full_description)
- Change list styles 
- Add list icon
- Define styles to heading (h2)

## Container
- Fix max-width of the container product-page to improve readability in text paragraphs (desktop)
- Fix mobile spacing
- Fix image spacing
- Fix button invisible div

### CODE REVIEW

File: react/components/ProductDescription.js [hunk 1]
```diff
@@ -28,11 +28,15 @@ const SLIDER_CENTER_MODE_EXTRA_SMALL_MOBILE = false
...
   elements: {
-    h4: ({ children }) => <div className="f5">{children}</div>,
+    h2: ({ children }) => <h2 className="f4-ns f5 fw7">{children}</h2>,
     ul: ({ children }) => <ul className="f5 list pl0 pr3">{children}</ul>,
     li: ({ children }) => (
-      <li className="lh-copy flex flex-row">
-        <div className="f7 ma1">&#9643;</div>
+      <li className="mb3 lh-copy flex flex-row">
+        <div className="mr3"><svg width="12" height="12" viewBox="0 0 12 12" fill="none">
+              <react width="12" height="12" fill="white"/>
```
Comment (ArturPimentel) [inline react/components/ProductDescription.js]: Shouldn't this tag be `<rect />`?

File: react/components/ProductDescription.js [hunk 2]
```diff
@@ -28,11 +28,15 @@ const SLIDER_CENTER_MODE_EXTRA_SMALL_MOBILE = false
...
+              <path d="M8 3H4C3.44772 3 3 3.44772 3 4V8C3 8.55228 3.44772 9 4 9H8C8.55228 9 9 8.55228 9 8V4C9 3.44772 8.55228 3 8 3Z" stroke="#3F3F40" stroke-width="1.4" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (ArturPimentel) [inline react/components/ProductDescription.js]: The HTML attributes with dashes `-` should be written in *camelCase* in JSX:
`stroke-width => strokeWidth`

File: react/components/ProductDescription.js (hunk 1 above)
Comment (ArturPimentel) [reply react/components/ProductDescription.js]: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/rect

Comment (ntmedina) [reply react/components/ProductDescription.js]: yes, sure! Probably some typing error, I'll fix it. thxx

**Pull request merged**